### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Alexandre Philibert + Valentin Ricard

### DISCUSSION THREAD

**Discussion by knownasred** [heig-vd-dai-course/heig-vd-dai-course#66] [Show and tell]
Our group is composed of Alexandre Philibert + Valentin Ricard (me)

**Comment by ludelafo** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by knownasred** [reply]
We've settled on something ^^
In a kubernetes cluster, many files are lookalikes, and only changes in minor ways. The objective is to offer a CLI to template some documents with pre-defined values.

**Reply by ludelafo** [reply]
Perfect! Thank you :)

**Comment by AlexandrePhilibert** [comment]
Link to the repo : https://github.com/AlexandrePhilibert/dai-labo-1
Hash of the last commit : [0f5b6464da14db61bb9f6ea11013a8615c866e72](https://github.com/AlexandrePhilibert/dai-labo-1/tree/0f5b6464da14db61bb9f6ea11013a8615c866e72)
